receiver/googlecloudspannerreceiver/internal/statsreader/intervalstatsreader.go

```diff
@@ -82,6 +84,11 @@ func (reader *intervalStatsReader) Read(ctx context.Context) ([]*metadata.Metric
 		if err != nil {
 			return nil, err
 		}
+		if reader.hideTopnLockstatsRowrangestartkey && stmt.statement.SQL == "SELECT * FROM SPANNER_SYS.LOCK_STATS_TOP_MINUTE WHERE INTERVAL_END = @pullTimestamp ORDER BY INTERVAL_END DESC, LOCK_WAIT_SECONDS DESC LIMIT @topMetricsQueryMaxRows" {
```

Comment:
Is the sql query a common constant?  Otherwise, this seems incredibly specific...

Reply:
Good point!
While the SQL query is a string declared in receiver/googlecloudspannerreceiver/internal/metadataconfig/metadata.yaml -- it is subject to change at a later point of time.

From the same metadata file above, the metric name "top minute lock stats" is not subject to change. Hence I have instead added that to the if condition above